.github/workflows/e2e.yml

```
@@ -47,48 +47,41 @@ jobs:
-          path: fiftyone/server/static
-          key: app-static-${{ runner.os }}-${{ steps.app-tree-hash.outputs.hash }}
-
-      - name: Cache Node Modules
-        id: app-node-cache
-        uses: actions/cache@v5
-        with:
-          path: |
-            app/node_modules
-            app/.yarn/cache
-          key: node-modules-${{ hashFiles('app/yarn.lock') }}
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)